src/service/cid-impl.js

```diff
@@ -305,9 +305,11 @@ export class Cid {
           if (scope) {
             apiKeyMap[scope] = API_KEYS[clientName];
           } else {
-            user().error(
+            user().warn(
               TAG_,
-              `Unsupported client for Google CID API: ${clientName}`
+              `Unsupported client for Google CID API: ${clientName}. Please remove or correct ${
```

Comment:
nit, move `${optInMeta./*OK*/outerHTML}` to same line. line break before `Please remove...`

Reply:
hm.. this was auto-formatted.